### PR TITLE
Update punctuation.md

### PR DESCRIPTION
**Pull request by ana-sorokina**
Add punctuation text rules

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Nice improve.

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [ ] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Comment by ana-sorokina** [comment]
I will need some help with list formatting and adding image examples

File: website/docs/content/punctuation/punctuation.md [hunk 1]
```diff
@@ -4,11 +4,196 @@ title: Punctuation and special symbols
...
+### Em dash
 
 | Symbol       | HTML entity        |
 | :----------: | :----------------: |
 | &mdash;      | `&mdash;`          |
 
 * Use sparingly in sentences in place of a comma or colon
 * Omit the spaces on both sides of the dash
+
+![](/content/punctuation/static/emdash.jpg)
```
Comment (j-mnizhek) [inline website/docs/content/punctuation/punctuation.md:16]: Nastya, can you please use png format for illustrations? Png saves image quality better. You also can compress them using the [TinyCanvas Figma plugin](https://www.figma.com/community/plugin/794829197801903069/tinycanvas) when exporting from Figma.

**Comment by j-mnizhek** [comment]
> I will need some help with list formatting and adding image examples

Sure! We will reach out to you a little bit later

**Review by sheila-semrush — APPROVED** [review]

**Comment by sheila-semrush** [comment]
I'll merge this to master so that Nastya can preview the page

**Pull request merged**